src/pages/account-restore.tsx

```diff
@@ -121,9 +124,9 @@ export const AccountRestore = (_: Props) => {
 						<p class="username">{profile?.username}</p>
 					</div>
 					<div class="btns">
-						<a class="btn btn-light" href={ACCOUNT}>
+						<Link className="btn btn-light" to={ACCOUNT}>
```

Comment:
I am a bit confused, don't you want to use `FlexLink` here and in other places instead of `Link`?

Reply:
https://github.com/swarmcity/terminal-ts/pull/42#discussion_r883379546